hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/properties/MirrorNodeEvmProperties.java

```diff
@@ -141,12 +138,13 @@ public int maxGasRefundPercentage() {
     @Getter
     @RequiredArgsConstructor
     public enum HederaNetwork {
-        MAINNET(unhex("00")),
-        TESTNET(unhex("01")),
-        PREVIEWNET(unhex("02")),
-        OTHER(unhex("03"));
+        MAINNET(unhex("00"), HederaChainId.MAINNET),
+        TESTNET(unhex("01"), HederaChainId.TESTNET),
+        PREVIEWNET(unhex("02"), HederaChainId.PREVIEWNET),
+        OTHER(unhex("03"), HederaChainId.OTHER);
 
         private final byte[] ledgerId;
+        private final HederaChainId hederaChainId;
```

Comment:
We now only have one property, but we still have two enums for each public network. Why do we need the `HederaChainId` enum? We should add the chain ID directly to `HederaNetwork` and remove `HederaChainId`.

```suggestion
        private final String chainId;
```